spec/javascript/components/lease_ups/LeaseUpApplicationsPage.test.js

```diff
@@ -160,4 +160,49 @@ describe('LeaseUpApplicationsPage', () => {
     // if submit wasn't clicked we shouldn't trigger an API request
     expect(mockCreateFieldUpdateComment).not.toHaveBeenCalled()
   })
+
+  test('updates substatus and last updated date on status change', async () => {
+    // Get the status last updated date before we change it, to verify that it changed.
+    const dateBefore = wrapper.find(rowSelector).first().find('PrettyTime').text()
+
+    // Change status to Appealed and open up the modal
+    act(() => {
+      wrapper
+        .find(rowSelector)
+        .first()
+        .find('Select')
+        .instance()
+        .props.onChange({ value: 'Appealed' })
+    })
+    wrapper.update()
+    expect(wrapper.find(StatusModalWrapper).props().isOpen).toBeTruthy()
```

Comment:
nit: can you copy this and put it around line 167 with ...toBeFalsy()? just to check it's not open at first then it opens

Reply:
we test the modal opening and closing in the test above so seems a bit redundant. I can remove this line here, mostly had it to double check that it was open before trying to edit it.

Reply:
just went ahead and removed this line